### PR TITLE
Use link_to_directory helper for gems letter directory

### PR DESCRIPTION
**Pull request by farukaydin**
Gems page letter directory is provided by view without using helper. However there is a helper method called link_to_directory which is built for this exact purpose but somehow is not used anymore.
I modified and used link_to_directory method again, with test.

cc // @qrush

### CODE REVIEW

**Comment by sferik** [comment]
:+1:

File: app/helpers/rubygems_helper.rb [hunk 1]
```diff
@@ -12,7 +12,9 @@ def link_to_page(text, url)
   end
 
   def link_to_directory
-    ("A".."Z").map { |letter| link_to(letter, rubygems_path(:letter => letter)) }.join
+    ("A".."Z").map do |letter|
+      link_to(letter, rubygems_path(:letter => letter), :class => "gems__nav-link")
```
Comment (qrush) [inline app/helpers/rubygems_helper.rb]: Nice! Let's use the 1.9+ hash syntax here too if we're moving the lines around.

Comment (farukaydin) [inline app/helpers/rubygems_helper.rb]: @qrush Opss sorry. I changed the other one in test but forget this one. @sferik merged this PR. Should I send another PR for that ?

**Comment by farukaydin** [comment]
@sferik @qrush thanks for super-fast response :+1:

File: app/helpers/rubygems_helper.rb (hunk 1 above)
Comment (sferik) [inline app/helpers/rubygems_helper.rb]: No need. I’ll clean it up in `master`.
